administrator/language/de-DE/com_contact.ini

```
@@ -69,6 +69,7 @@ COM_CONTACT_FIELD_ICONS_MISC_LABEL="Symbol für weitere Informationen"
 COM_CONTACT_FIELD_ICONS_MOBILE_LABEL="Mobilsymbol"
 COM_CONTACT_FIELD_ICONS_SETTINGS_LABEL="Einstellungen"
 COM_CONTACT_FIELD_ICONS_TELEPHONE_LABEL="Telefonsymbol"
+COM_CONTACT_FIELD_ICONS_WEBPAGE_LABEL="Individuelles Webseiten-Symbol"
```

```suggestion
COM_CONTACT_FIELD_ICONS_WEBPAGE_LABEL="Benutzerdefiniertes Website-Symbol"
```
fyi @max123kl 
Here webpage means the website.
We once defined that webpage is a single page of a website.